src/Equinor.ProCoSys.IPO.Query/GetOutstandingIpos/GetOutstandingIposForCurrentPersonQueryHandler.cs

```diff
@@ -93,30 +146,30 @@ public class GetOutstandingIposForCurrentPersonQueryHandler : IRequestHandler<Ge
             }
         }
 
-        private static bool UserWasInvitedAsPersonParticipant(Invitation invitation, Guid currentUserOid)
-            => invitation.Participants.Any(p => 
+        private static bool UserWasInvitedAsPersonParticipant(InvitationDto invitation, Guid currentUserOid)
+         => invitation.Participants.Any(p =>
                 p.AzureOid == currentUserOid
                 && p.FunctionalRoleCode == null
-                &&((!p.SignedAtUtc.HasValue 
-                && p.Organization != Organization.Supplier 
+                && ((!p.SignedAtUtc.HasValue
+                && p.Organization != Organization.Supplier
                 && p.Organization != Organization.External
-                && p.SortKey != 1) 
+                && p.SortKey != 1)
                 || (p.SortKey == 1 && invitation.Status == IpoStatus.Completed)));
 
-        private static bool UserWasInvitedAsPersonInFunctionalRole(Invitation invitation, IEnumerable<string> currentUsersFunctionalRoleCodes)
+        private static bool UserWasInvitedAsPersonInFunctionalRole(InvitationDto invitation, IEnumerable<string> currentUsersFunctionalRoleCodes)
         {
-            var functionalRoleParticipantCodesOnInvitation = 
+            var functionalRoleParticipantCodesOnInvitation =
```

Comment:
rearrange the last part to ease reading
![image](https://user-images.githubusercontent.com/19545528/193746294-04e407d3-e5d4-4ebc-b73e-f31d2670e667.png)